src/pymanopt/optimizers/nelder_mead.py

```diff
@@ -12,14 +12,12 @@
 def compute_centroid(manifold, points):
     """Compute the centroid of `points` on the `manifold` as Karcher mean."""
```

Comment:
Suggestion of @NicolasBoumal:
```suggestion
    """Compute the centroid of `points` on the `manifold` as intrinsic mean."""
```